src/utils/helpers.js

```diff
@@ -41,7 +41,10 @@ function remotelyExec (machineName, zone, command, cb) {
       const trimmed = String.prototype.trim(data)
       if (trimmed) {
         console.log(`stdout: ${trimmed}`)
+      } else {
+        console.log('untrimmed stdout', (data) ? data.toString() : null)
```

Comment:
If `trimmed` is falsey wouldn't there be nothing to log? Also, should we check that `data` is truthy and *then* try to call `trim()` on `data`?

Reply:
actually, not always, check this out 
```js

    builder.stdout.on('data', (data) => {
      if (data) {
        const trimmed = String.prototype.trim(data.toString())

        if (trimmed) {
          console.log(`stdout: ${trimmed}`)
        } else {
          console.log('untrimmed stdout', data.toString())
        }
        
        output += data
      }
    })

    builder.stderr.on('data', (data) => {
      if (data) {
        if (data == '.') return
        const trimmed = String.prototype.trim(data)
        if (trimmed) {
          console.log(`stderr: ${trimmed}`)
        } else {
          console.log('untrimmed stderr', (data) ? data.toString() : null)
        }
      }
    })
```

this still prints out `untrimmmed stderr` and `untrimmed stdout` 
welcome to JS :D

Reply:
Need to use `data.trim()` instead of `String.prototype.trim(data)`

If you want to use the prototype you'll need to use `call` -> `String.prototype.trim.call(data`